arcgis-ios-sdk-samples/Features/Display subtype feature layer/DisplaySubtypeFeatureLayer.storyboard

```diff
@@ -0,0 +1,244 @@
+                                                    <rect key="frame" x="16" y="11.5" width="172.5" height="21"/>
+                                                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                                    <nil key="textColor"/>
+                                                    <nil key="highlightedColor"/>
+                                                </label>
+                                                <switch opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" contentHorizontalAlignment="center" contentVerticalAlignment="center" on="YES" translatesAutoresizingMaskIntoConstraints="NO" id="Q2g-ka-lq7" userLabel="Toggle Renderer Switch">
+                                                    <rect key="frame" x="349" y="6.5" width="51" height="31"/>
+                                                    <connections>
+                                                        <action selector="rendererSwitchAction:" destination="HLD-kJ-COl" eventType="valueChanged" id="X4z-lq-rhm"/>
+                                                    </connections>
+                                                </switch>
+                                            </subviews>
+                                            <constraints>
+                                                <constraint firstAttribute="trailing" secondItem="Q2g-ka-lq7" secondAttribute="trailing" constant="16" id="4W3-Bn-VXu"/>
+                                                <constraint firstItem="RT5-P8-WU4" firstAttribute="centerY" secondItem="slj-Q0-13T" secondAttribute="centerY" id="Duz-vA-3OB"/>
+                                                <constraint firstItem="RT5-P8-WU4" firstAttribute="leading" secondItem="slj-Q0-13T" secondAttribute="leading" constant="16" id="awy-7C-Wel"/>
+                                                <constraint firstItem="Q2g-ka-lq7" firstAttribute="centerY" secondItem="slj-Q0-13T" secondAttribute="centerY" id="iAP-LU-6bB"/>
+                                            </constraints>
+                                        </tableViewCellContentView>
+                                    </tableViewCell>
+                                </cells>
+                            </tableViewSection>
+                            <tableViewSection headerTitle="Sublayer Minimum Scale" id="Pgz-VK-Ihy" userLabel="Sublayer Minimum Scale">
+                                <cells>
+                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="default" indentationWidth="10" id="xal-8G-oUO">
```

Comment:
Please use a right detail cell for this and the Minimum Scale cell.

Reply:
@philium What is a right detail cell and how can I implement it?

Reply:
It is one of the non-custom cell styles provided by UIKit:

<img width="258" alt="image" src="https://user-images.githubusercontent.com/2257493/75716966-16bb4980-5c85-11ea-9cd3-724ecb58b5bb.png">

It causes the cell to have two labels, one on the left and the other on the right.